meteor-app/imports/ui/layouts/attendance.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Sidebar, Menu, Icon } from 'semantic-ui-react'
 import { Switch, Route } from 'react-router-dom'
 import NotFound from '/imports/ui/not-found'
-import MemberAddContainer from '/imports/ui/member/member-add-container'
+import MemberAddContainer from '/imports/ui/member/add/member-add-container'
```

Comment:
Let's not go any deeper with the folder structure - we are using composite names already to help us avoid that